content/webapp/components/Quote/Quote.tsx

```diff
@@ -38,7 +38,7 @@ const Quote: FunctionComponent<Props> = ({
         <PrismicHtmlBlock html={text} />
       </Space>
       {citation && (
-        <footer className="quote__footer flex">
+        <footer className="quote__footer" style={{ display: 'flex' }}>
```

Comment:
`quote__footer` doesn't seem to be declared anywhere so I'd remove it. (investigating quote classes now but this one seems undeclared)

Reply:
([Found an example here](https://wellcomecollection.org/exhibitions/WduTricAAN7Mt8yY) and removed the class without any consequence)

Reply:
Removed. I think there's more tidy-up we can do in the Quote styles, but I didn't want to put any more in this PR.

Reply:
100%, I'm bringing it up in the FE catchup as [it appears the quotes styling is broken](https://wellcome.slack.com/archives/C3N7J05TK/p1681202766701939).  Just this one wasn't declared at all and would've gone under the radar!